### PR TITLE
add a MAINTAINERS file

### LINKED ISSUE #83: Add a MAINTAINERS file

### PR DESCRIPTION
**Pull request by nkubala**
fixes #83 

cc @dlorenc @r2d4 @aaron-prindle @priyawadhwa @cftorres @abbytiz @duggelz

### CODE REVIEW

File: MAINTAINERS [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+			"mattrickard",
+			"nkubala"
+		]
+
+	[Org."Alumni"]
+
+	# People who have contributed to the project in the past.
+
+		people = [
+			"abbytisdale",
```
Comment (abbytiz) [inline MAINTAINERS]: I think this should be abbytiz if it's supposed to by my github username :)

Comment (nkubala) [reply MAINTAINERS]: It's actually just the key into the `people` references section below, which has your name, email, and username. Check those and make sure they're right though :+1:

File: MAINTAINERS [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+	[Org."Core maintainers"]
+
+	# Current active contributors to the project.
+
+		people = [
+			"aaronprindle",
+			"danlorenc",
+			"mattrickard",
+			"nkubala"
```
Comment (sharifelgamal) [inline MAINTAINERS]: This should be nickkubala

Comment (nkubala) [reply MAINTAINERS]: thanks, fixed

File: MAINTAINERS (hunk 1 above)
Comment (abbytiz) [reply MAINTAINERS]: Oh, LGTM

**Comment by duggelz** [comment]
Is this a format that is parsed by something?  Otherwise, maybe we should have a GitHub CODEOWNERS file instead? https://help.github.com/articles/about-codeowners/

**Comment by nkubala** [comment]
> Is this a format that is parsed by something? Otherwise, maybe we should have a GitHub CODEOWNERS file instead? https://help.github.com/articles/about-codeowners/

This would allow us to write some validation scripts later on if we wanted to, but I didn't realize Github had CODEOWNERS built in :)

**Pull request closed without merging**